style.css

```diff
@@ -173,6 +175,57 @@ button::-moz-focus-inner {
   padding: 3px;
 }
 
+.separator {
+  height: 0px;
+  width: 100%;
+  border-top:1px solid var(--button-shadow);
+  border-bottom: 1px solid var(--button-highlight);
+}
+
+button.menu-bar {
+  box-shadow: none;
+  min-width: 0;
+  padding: 0 6px;
+}
+
+button.menu-bar:focus + .menu {
```

Comment:
One thing to consider is that in order to be keyboard accessible, the focus will need to move from `button.menu-bar` to one of the `li`s.

You won't be able to solve this with CSS alone, but instead will need to rely on `aria-expanded` (instead of `:focus`), which will have to be toggled with JavaScript. (JavaScript provided by the user, not by us).

One exit criteria is that we should be able to render https://reacttraining.com/reach-ui/menu-button/ as-is by just including `98.css`. See the discussion in #32. 

I'll probably include something to this effect in the readme since it keeps popping up.